addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/SourceCodeDisclosureScanRuleUnitTest.java

```diff
@@ -107,6 +109,29 @@ void givenSQLAndPhpCodeThenOnlyOneAlertRaised() {
         assertThat(alertsRaised.size(), is(1));
     }
 
+    @Test
+    void shouldNotRaiseAlertOnCssResponse() throws Exception {
+        // Given
+        msg.getRequestHeader().setURI(new URI("https://www.example.com/assets/styles", true));
```

Comment:
- These messages should be setup in a way that they would raise an alert, otherwise the tests will still pass even if the check is not correct.
 - Should be added a test for the CSS request condition too.

Reply:
_These messages should be setup in a way that they would raise an alert, otherwise the tests will still pass even if the check is not correct._

@thc202 I tried to put the message, but now that we are looking for "header", the body scan is not getting executed. So, I didn't add the test case.

Reply:
That's ok, it's not about now but for future changes. We want the test to fail when the header check is wrong (e.g. while doing a refactoring an issue was introduced) so it needs the body also.

Reply:
Still needs the test for the request condition.

Reply:
@thc202 I am trying to identify which exact java regex pattern is causing the failure here. Interestingly my local ZAP no more doing any source code disclosure alerts (even after I reverted code in my local). 

@psiinon I see some error/exception with these lines, could this be causing the sourcecodedisclosure rule NOT run? please advise.
```
27691 [ZAP-PassiveScanner] ERROR org.zaproxy.zap.extension.pscanrulesAlpha.CacheableScanRule - An error occurred while checking a URI [https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?version=3&consumer_key=40249-e88c401e1b1f2242d9e441c4&locale_lang=en-US&region=US&count=30] for cacheability
java.lang.NumberFormatException: For input string: "900,public"
```

Reply:
Could you provide more details?

That logging message is unrelated to this scan rule.

Reply:
Before scan this is the error I see in my ZAP console.
```
[ZAP-PassiveScanner] ERROR org.zaproxy.zap.extension.pscan.PassiveScanThread - Scanner Wappalyzer Scanner (Tech Detection) failed on record 6 from History table: GET http://localhost:3000/socket.io/?EIO=4&transport=polling&t=NeNvg74&sid=yZpJG6fLwF6Ayn3yAAAE
org.jsoup.select.Selector$SelectorParseException: Did not find balanced marker at 'src*='

When I initiated "spider", there are whole sorts of these errors showing up my console. If these are not related to my issue, I will ignore.
@thc202 

```

Reply:
Right, that's unrelated, if you update the Wappalyzer add-on (built from source or reinstall the one from marketplace) those errors should go away.

Reply:
The messages were not changed yet.